application/src/main/java/com/sanction/lightning/facebook/FacebookService.java

```diff
@@ -95,6 +97,24 @@ public FacebookUser getFacebookUser() {
   }
 
   /**
+   * Published a file to facebook using the restFB api.
+   *
+   * @return A boolean to denote that the function worked.
+   */
+  public FacebookPhoto publishToFacebook(byte[] inputBytes, String fileName, String message) {
+    FacebookPhoto response;
+
+    try {
+      response = client.publish("me/photos", FacebookPhoto.class,
+              BinaryAttachment.with(fileName, inputBytes), Parameter.with("message", message));
```

Comment:
Yes but I facebook renames it anyway. I included it because restFB requires a file name and it's available when the request comes in so might as well.